java/pattern/constraint/TypeConstraint.java

```diff
@@ -466,7 +466,17 @@ public boolean isOwns() {
 
         @Override
         public String toString() {
-            return OWNS.toString() + SPACE + attributeType + (isKey ? "" + SPACE + IS_KEY : "");
+            StringBuilder syntax = new StringBuilder();
+            syntax.append(OWNS).append(SPACE);
+            if (!attributeType.label().isPresent()) syntax.append(attributeType);
+            else syntax.append(attributeType.label().get().label());
+            if (isKey) syntax.append(SPACE).append(IS_KEY);
+            if (overriddenAttributeType!=null) {
```

Comment:
i think we prefer the more spaced out style with spaces on either side of the inequality, or whichever intelliJ's auto format performs

Reply:
If statements duly socially distanced.

Reply:
> i think we prefer the more spaced out style with spaces on either side of the inequality, or whichever intelliJ's auto format performs

Do we? :D I think we prefer to make code as compact as possible (without going over 120 character width limit)